src/mrpro/data/KHeader.py

```diff
@@ -12,7 +12,7 @@
 import torch
 
 from mrpro.data import enums
-from mrpro.data.AcqInfo import AcqInfo, mm_to_m, ms_to_s
+from mrpro.data.AcqInfo import AcqInfo, m_to_mm, mm_to_m, ms_to_s, s_to_ms
```

Comment:
We could move the conversions to utils.conversions and add some for hz/Tesla/ppm ?
(And amy other common scaling factors.)

Reply:
done in #425